src/pybind/mgr/openattic/module.py

```diff
@@ -24,8 +23,17 @@ def __init__(self, *args, **kwargs):
         super(Module, self).__init__(*args, **kwargs)
 
     def serve(self):
-        cherrypy.config.update({'server.socket_host': '0.0.0.0',
-                                'server.socket_port': 8080,
+        server_addr = self.get_localized_config('server_addr', '::')
```

Comment:
Shouldn't we "fallback" to cluster-wide configuration before return the default value (`self.get_config('server_addr')`), like described on ceph-mgr dashboard documentation? http://docs.ceph.com/docs/master/mgr/dashboard/

Reply:
get_config returns the cluster-wide config only, get_localized_config checks for cluster-specific configs and falls back to cluster-wide if necessary, see https://github.com/ceph/ceph/blob/master/src/pybind/mgr/mgr_module.py#L198.